modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -937,6 +938,240 @@ m|WARNING
 m|DEPRECATION
 |===
 
+[#_security_notifications]
+== `SECURITY` category
+
+Notifications in the security category indicates that the result of the query or command might have a potential security issue.
+Make sure that the behaviour is what was intended.
+
+[#_neo_clientnotification_security_commandhadnoeffect]
+=== Neo.ClientNotification.Security.CommandHadNoEffect
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Security.CommandHadNoEffect
+|Title
+a|`<command>` has no effect.
```

Comment:
How do we want to handle details in the titles? I added `Full title of the returned code` in the examples, is that a good way?

Reply:
I think that works, but would like to hear what @renetapopova thinks as well